source/Willow-Bootstrap/BootstrapComponentSupplier.class.st

```diff
@@ -214,6 +214,14 @@ BootstrapComponentSupplier >> tabsBuilder [
 	^ self subclassResponsibility
 ]
 
+{ #category : #Supplying }
+BootstrapComponentSupplier >> timeFieldApplying: aComponentCommand [
+
+	^ self
+		singleLineTextFieldApplying:
```

Comment:
Here will be better to use a `TimeFieldWebView` so you can use the specific protocol like `setTimeTo:` and its friends. If you need it to be a basic text field because the time field in bootstrap looks ugly you can send beTextInput as a command.